### ISSUE TITLE
hit boxes of diagonal slopes are weird 

### ISSUE THREAD

**Issue by AsumFace** [KaylebJay/tunnelers-bugz#7]
The hit box of `underch:slope_*_outer_cut_half` extends too far beyond the visual model, combining it with `underch:slope_*_inner_cut_half_raised` (for example for a roof) results in a visually flat but collision-wise jagged surface that is difficult to walk on.
Same situation with `underch_*_outer_cut` and `underch:slope_*_inner_cut`.

**Comment by KaylebJay** [comment]
Agreed. However, this shouldn't be just for underch - this is a direct problem in moreblocks, I'd say.
Using shift and jumping isn't quite good enough anymore.

**Comment by AsumFace** [comment]
you're absolutely right, moreblocks generates them :man_facepalming:
